README.md

```diff
@@ -46,4 +46,4 @@ would find it with the following configuration snippet:
 Server = file:///path/to/output
 ```
 
-ART keeps a cache file (`.art-cache`) in its current working directory to speed up incremental rebuilds.
+ART keeps a cache file (`.art-cache`) in it's current working directory to speed up incremental rebuilds.
```

Comment:
Was correct before.

Reply:
Don't understand your comment.

Reply:
What I mean is that `its` is correct here. When you expand `it's` to `it is` in your mind, it should be clearer that it doesn't make sense that way.

Reply:
Oh, grammar. GG German teaching me proper English grammar.